### PR TITLE
Add Single Templates for generic scenario and extend tests with Controller scenario

### PR DESCRIPTION
**Pull request by jakubschwan**

### CODE REVIEW

File: test-cloud/src/main/resources/templates/jbpm/rhpam70-businesscentral-monitoring.yaml [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+kind: Template
```
Comment (jhrcek) [inline test-cloud/src/main/resources/templates/jbpm/rhpam70-businesscentral-monitoring.yaml]: Do we want to commit templates to our testing repo, as opposed to download them with every prod build?

File: test-cloud/pom.xml [hunk 2]
```diff
@@ -156,6 +160,15 @@
     </profile>
     <profile>
       <id>jbpm</id>
+      <properties>
+        <kie.app.template.workbench>${templates.jbpm.dir.url}/rhpam70-businesscentral.yaml</kie.app.template.workbench>
+        <kie.app.template.workbench-monitoring>${templates.jbpm.dir.url}/rhpam70-businesscentral-monitoring.yaml</kie.app.template.workbench-monitoring>
+        <kie.app.template.smartrouter>${templates.jbpm.dir.url}/rhpam70-smartrouter.yaml</kie.app.template.smartrouter>
+        <kie.app.template.controller>${templates.jbpm.dir.url}/rhpam70-controller.yaml</kie.app.template.controller>
+        <kie.app.template.kie-server.database>${templates.jbpm.dir.url}/rhpam70-kieserver-postgresql.yaml</kie.app.template.kie-server.database>
```
Comment (sutaakar) [inline test-cloud/pom.xml]: This template is available as official one, I think we should take it from there.

File: test-cloud/pom.xml [hunk 3]
```diff
@@ -156,6 +160,15 @@
...
+        <kie.app.template.kie-server.database.external>${templates.jbpm.dir.url}/rhpam70-kieserver-externaldb.yaml</kie.app.template.kie-server.database.external>
```
Comment (sutaakar) [inline test-cloud/pom.xml]: Same as above.

File: test-cloud/src/main/resources/templates/jbpm/rhpam70-businesscentral-monitoring.yaml (hunk 1 above)
Comment (jhrcek) [reply test-cloud/src/main/resources/templates/jbpm/rhpam70-businesscentral-monitoring.yaml]: Ok, thanks for explanation.

Comment (jakubschwan) [reply test-cloud/src/main/resources/templates/jbpm/rhpam70-businesscentral-monitoring.yaml]: And "signle" template is used if there isn't a prod one.

**Comment by sutaakar** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

**Comment by jakubschwan** [comment]
retest this please

**Pull request merged**